navit/navit_layout_car_shipped.xml

```diff
@@ -444,15 +445,16 @@
 			<text text_size="5"/>
 		</itemgra>
 		<itemgra item_types="poly_sports_track" order="10-">
-			<polygon color="#b6554e"/>
+			<polygon color="#bb885599"/>
```

Comment:
This again is something where transparency "should not" be required as this "should not" overlap anything. However do you have any good example on where you need transparency on poly_sports_track? Because I think most probably this are only mapping mistakes in OSM.

Reply:
There are many good examples.
Here are a few to make my point:
132635227
420091556
179372237

Reply:
According to https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dtrack it's unclear how those things should be tagged and rendered. They are allowed either as line or as area. Since multipolygons are always areas, Id support both. I think adding a polygon and a polyline could be a good compromise. However loop tracks are then as up to now rendered as areas albeit they shouldn't as long as area=yes is missing.
think of:
<polygon color="#bb8855"/>
<polyline color="#bb8855"/>

We need to address the "closed loop but no area" thing in maptool by giving the non area tracks their own map id. E.g. sports_track without "poly_" and decide on the presence of area= tag which one to use.

But still I think the use in https://www.openstreetmap.org/way/179372237 is a mapping error. As it is somehow used like a landuse not giving the outline of the actual track nor the track itselve.
So removing the transparency should be fine.